paymentsheet/src/main/java/com/stripe/android/paymentsheet/addresselement/AddressLauncher.kt

```diff
@@ -158,30 +157,42 @@ internal class AddressLauncher internal constructor(
                 defaultValues,
                 allowedCountries,
                 buttonTitle,
-                phone,
+                additionalFields,
                 title,
                 googlePlacesApiKey,
                 checkboxLabel
             )
         }
     }
 
+    /**
+     * @param phone Configuration for the field that collects a phone number. Defaults to
+     * [FieldConfiguration.OPTIONAL]
+     * @param checkboxLabel The label of a checkbox displayed below other fields. If nil, the
+     * checkbox is not displayed. Defaults to null
+     */
     @Parcelize
-    enum class AdditionalFieldsConfiguration : Parcelable {
-        /**
-         * The field is not displayed.
-         */
-        HIDDEN,
-
-        /**
-         * The field is displayed but the customer can leave it blank.
-         */
-        OPTIONAL,
-
-        /**
-         * The field is displayed and the customer is required to fill it in.
-         */
-        REQUIRED
+    data class AdditionalFieldsConfiguration(
```

Comment:
Updating the structure of `AddtionalFieldsConfiguration` to match iOS

Reply:
Do we need a `@JvmOverloads` for Java?